sydent/threepid/bind.py

```diff
@@ -69,6 +98,10 @@ def addBinding(self, medium, address, mxid):
 
         self.sydent.pusher.doLocalPush()
 
+        signer = Signer(self.sydent)
+        sgassoc = signer.signedThreePidAssociation(assoc)
+        return sgassoc
```

Comment:
if we're now returning an sgassoc, we need to update the pydoc.

also, why are we returning an sgassoc, given we haven't changed anywhere which calls addBinding?

Reply:
It always returned an sgassoc but it got lost in https://github.com/matrix-org/sydent/commit/03882e5500e8a7b18ff135a57f0004180c04eb91 which was causing Riot to error when adding an email address (I guess dinsic swallowed the error).